eng/scripts/build.ps1

```diff
@@ -25,6 +25,10 @@ function Process-Sdk ($path) {
             $outputFolder = $path
         }
         autorest --use=$autorestVersion --go --track2 --go-sdk-folder=$root --output-folder=$outputFolder --file-prefix="zz_generated_" --clear-output-folder=false $autorestPath
+        if ($LASTEXITCODE) {
+            Write-Host "##[command]Error running autorest.go"
```

Comment:
https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/pipelines/scripts/logging-commands.md#formatting-commands you can use `##[error]Error message` will give a nice red error.